artemis-core-client/src/main/java/org/apache/activemq/artemis/core/config/federation/FederationDownstreamConfiguration.java

```diff
@@ -40,23 +41,30 @@ public TransportConfiguration getUpstreamConfiguration() {
       return upstreamConfiguration;
    }
 
+   private void stripParam(Map<String, Object> params, String name) {
+	   Object oldValue = params.remove(name);
+	   if (oldValue != null) {
+		   ActiveMQClientLogger.LOGGER.ignoredParameterForDownstreamFederation(name);
+	   }
```

Comment:
there are 8 checkstyle violations because lines 45-48 contain tab characters

Reply:
I just saw the build report too. I'll soon fix that...